YOLO.xcodeproj/project.pbxproj

```diff
@@ -379,7 +359,7 @@
 				ASSETCATALOG_COMPILER_INCLUDE_ALL_APPICON_ASSETS = NO;
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 0;
-				DEVELOPMENT_TEAM = 3MR4P6CL3X;
+				DEVELOPMENT_TEAM = "";
```

Comment:
why this change?

Reply:
I left it blank because when users build the project locally, they need to change it to their own team. If you want to change it back to the original team, please let me know.

Reply:
```suggestion
				DEVELOPMENT_TEAM = 3MR4P6CL3X;
```
Keep the Development team!